ci/pipelines/e2e-helpers.yml

```diff
@@ -380,83 +399,92 @@ jobs:
       put: slack-notification
       attempts: 10
       params:
-        channel: '#govuk-pay-starling'
+        channel: "#govuk-pay-starling"
         silent: true
-        text: ':red-circle: pay-endtoend failed post-merge e2e tests - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>'
+        text: ":red-circle: pay-endtoend failed post-merge e2e tests - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>"
         icon_emoji: ":concourse:"
         username: pay-concourse
     on_success:
       put: slack-notification
       attempts: 10
       params:
-        channel: '#govuk-pay-activity'
+        channel: "#govuk-pay-activity"
         silent: true
-        text: ':green-circle: pay-endtoend passed post-merge e2e tests and was pushed as a final release - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>'
+        text: ":green-circle: pay-endtoend passed post-merge e2e tests and was pushed as a final release - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>"
         icon_emoji: ":concourse:"
         username: pay-concourse
 
   - name: build-and-push-reverse-proxy-candidate
     plan:
       - in_parallel:
-        - get: reverse-proxy-git-release
-          trigger: true
-        - get: pay-infra-src
-          trigger: true
-        - get: pay-ci
+          - get: reverse-proxy-git-release
+            trigger: true
+          - get: pay-infra-src
+            trigger: true
+          - get: pay-ci
+          - task: parse-release-tag
+            file: pay-ci/ci/tasks/parse-release-tag.yml
+            input_mapping:
+              git-release: reverse-proxy-git-release
+          - load_var: release_number_tag
+            file: tags/release-number
```

Comment:
I think the `task` and the `load_var` here will need to come out of its parent `in_parallel`, because we need to wait for the `get: reverse-proxy-git-release` to complete before we can parse its tag. So just de-indent this and it should work.